src/js/providers/video-attached-mixin.js

```diff
@@ -15,6 +15,10 @@ const VideoAttachedMixin = {
         this.stopStallCheck();
         this.eventsOff_();
 
+        if (this.video && !this.video.paused && this.video.pause) {
```

Comment:
Why the check for `this.video.pause`?

Reply:
I'd rather not change provider behavior. This should stay scoped to the ads program controller - in setup.